nixos/modules/services/games/asphyxia-core.nix

```diff
@@ -0,0 +1,158 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.asphyxia-core;
+  opt = options.services.asphyxia-core;
+  configAsIni = generators.toINIWithGlobalSection {} {
+    globalSection = {
+          port = cfg.port;
+          bind = cfg.bind;
+          ping_ip = cfg.pingIp;
+          matching_port = cfg.matchingPort;
+          allow_register = cfg.allowRegister;
+          maintenance_mode = cfg.maintenanceMode;
+          enable_paseli = cfg.enablePaseli;
+    };
+    sections = {};
+  };
+
+  configIniFile = pkgs.writeText "config.ini" configAsIni;
+
+  configDir = pkgs.runCommand "createConfigDir" {} ''
+    mkdir -p $out
+    # create symlink to config file
+    ln -s ${configIniFile} $out/config.ini
+    cp -R ${corePlugins} $out/plugins
```

Comment:
Why do we need to cp here and can't symlink?

Reply:
can do